applications/DAOsign.md

```diff
@@ -0,0 +1,186 @@
+|    0e. | Article                   | We will publish an **article** that explains what was done as part of the grant. And we will publish a series of articles that explains how DAOsign works from a high-level perspective. The content of the articles will be consistent with the functions at this stage. |
+|     1. | Smart Contracts           | DAOsign Smart Contract repo that includes following components: Agreement Contract, EIP712/EIP2771 standard implementation on Ink!. These smart contracts can be deployed to any substrate chain with contracts-pallet.                                                   |                                                                                                                                                                                                                                           |
+|     2. | Gas Relayer               | Off-chain component for transaction relaying                                                                                                                                                                                                                              |
+
+
+### Milestone 2 - DAOsign Application integration
+
+* **Estimated Duration:** 2 weeks
+* **FTE:**  2
+* **Costs:** 7,800 USD
+
+
+| Number | Deliverable                     | Specification                                                                                                                                                     |
+|-------:|---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|    0a. | License                         | Apache 2.0                                                                                                                                                        |
+|    0b. | Documentation                   | We will provide both inline documentation of the code and a basic tutorial that explains how a user can                                                           |
+|    0c. | Testing Guide                   | Core functions will be fully covered by unit tests to ensure functionality and robustness.                                                                        |
+|     1. | DAOsign Ink! JS SDK             | We will publish a npm/yarn package with the logic how to interact with Smart Contract from JS                                                                     |
+|     2. | DAOsign Application Integration | Integrate DAOsign application using SDK. DAOsign application will be open sourced as well. DAOsign is written using ReactJS on frontend and Typescript on backend |
```

Comment:
For frontend deliverables we usually require the grantee to provide us with some wireframes.

Reply:
Is it ok to share wireframe/design here?

1. Adding toggle to allow store Agreement Proof on-chain (toggle above Sign Agreement button)

![Store proofs 0](https://github.com/w3f/Grants-Program/assets/1008882/40f89bfa-50fc-4944-8a64-af84337b713a)

2. Enable that toggle and select Parity network from dropdown

![Store proofs 1](https://github.com/w3f/Grants-Program/assets/1008882/87894c58-bc30-4e48-b74b-3eabfcad2779)

3. After user click `Sign Agreement` and execute transaction, we show `On-chain Proof` link to transaction on Information panel
![Store proofs 6](https://github.com/w3f/Grants-Program/assets/1008882/88dfed66-5f88-4658-88b4-8a6198060de9)

Reply:
Thanks @ramilexe these look good but if you could integrate them into the application that would be great, as this is the criteria we use for the contract and also helps us later in the process when evaluating milestones.

Reply:
Done